.github/ISSUE_TEMPLATE/standard-issue.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Standard Issue
+about: Standard information for PASS issues
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please answer the following questions for yourself before submitting an issue. 
```

Comment:
Shouldn't this line and the following be moved inside the Prerequisites section? They only appear to apply in that context.

Reply:
Agreed to make the change.